pulp_container/app/models.py

```diff
@@ -358,7 +447,7 @@ class Meta:
 
 
 class ContainerDistribution(
-    RepositoryVersionDistribution, AutoAddObjPermsMixin, AutoDeleteObjPermsMixin
+    RepositoryVersionDistribution, CreateGroupAndAddPermissionsMixin, AutoDeleteGroupsMixin
```

Comment:
Should we keep `AutoAddObjPermsMixin`, `AutoDeleteObjPermsMixin` everywhere as well?
So permission auto-assignment works for users who change the access policy.
And even without a change to access policy the permissions are cleaned properly from the groups created and assigned to by a user.
What do you think?

Can 2 hooks of the same type be performed? (2 after_delete and 2 after_create)?